src/utils/server-utils.ts

```diff
@@ -90,7 +90,7 @@ export const getPossibleDatesForLayer = (
     }
   };
 
-  return datesArray().map(d => d.displayDate);
+  return datesArray()?.map(d => d.displayDate);
```

Comment:
To be consistent, you should either retype the return of the function to be number[] | undefined, or return an empty array, eg.

`return datesArray()?.map(d => d.displayDate) ?? []`

Reply:
but the function should return undefined, based on this condition: https://github.com/WFP-VAM/prism-app/blob/master/src/components/MapView/index.tsx#L416

Reply:
included in 814af14d895085c45569af957e8a1ecc05fe1623